### ISSUE TITLE
Command line option to encrypt everything within a subscription with one command.

### ISSUE THREAD

**Issue by devanes** [plesk/letsencrypt-plesk#121]
In addition to issue #28 (Support for domain alliases) it would be great if there was a command line option to get a certificate to everything within a domain at once (all domains, aliases and subdomains, with and without www) without have to specify each individual (sub)domain.

Something like:
`plesk bin extension --exec letsencrypt cli.php -d domain.com -all --letsencrypt-plesk:plesk-secure-panel`

That way admins won't need to lookup all (sub)domains themselves.

**Comment by IronButterfly** [comment]
Yes, I can say that this feature is under developer's consideration and will be implemented. But not exactly in this way.
